### DISCUSSION TITLE
Vimix, keyboard and antimicroX

### DISCUSSION THREAD

**Discussion by zcbernard** [brunoherbelin/vimix#136] [Ideas]
AntimicroX associated with Vimix allows you to control the keyboard keys. It's very easy and pleasant. A PS2 style pad has 18 controls. I often use two pads simultaneously for 36 commands, but the keyboard only offers 24 inputs. Would it be possible to group the keyboard and the numeric keypad in the same window?
Bernard

**Comment by brunoherbelin** [comment]
Yes, sure! Do you prefer to have the two 'blocs' (all letters and all numbers) horizontally of vertically in the Input Mapping window? 
I was thinking that vertically would allow to have them all (keyboard, numpad, joystick, etc.) at once, but its not space optimal...

**Comment by zcbernard** [comment]
What if they were two independent and mobile blocks. We could have one or
the other (this is already the case) or both that we would arrange best in
space. If this is not possible, horizontal seems preferable to me

**Comment by zcbernard** [comment]
Hello, don't bother putting the number pad with the letter keys. Each can be programmed in its own display window, and they remain active. I'm sorry I didn't do the test correctly
Bernard

**Comment by brunoherbelin** [comment]
Aha, I see that there was a reason for a misunderstanding indeed! 
Because of the tic mark in the menu (keyboard / numpad / etc.), it looked like only one input group was active at a time:

![image](https://github.com/brunoherbelin/vimix/assets/60393189/adddce9d-a866-457b-8091-6d8c83186aa6)

This is indeed confusing as in effect all input mappings are active at all time (the menu is only to choose which one to show)

A minimal change that could help is to avoid suggesting that only one is selected by removing the 'tick' mark in the menu:

![image](https://github.com/brunoherbelin/vimix/assets/60393189/e253cf09-3e77-4da8-97a8-6dfca3a63d01)

Thanks for the feedback. I will for sure do this as it can avoid others to have the same misunderstanding.
We could also imagine better ways to show the input mapping, but this could be much more work...

**Comment by zcbernard** [comment]
and yes, it's very nice to deal with you.  always courteous, pushing the
debate and offering options.  thank you for all this.  Bernard
